app/src/main/kotlin/at/bitfire/davdroid/ui/AboutActivity.kt

```diff
@@ -81,14 +82,29 @@ class AboutActivity: AppCompatActivity() {
 
         binding.viewpager.adapter = TabsAdapter(supportFragmentManager)
         binding.tabs.setupWithViewPager(binding.viewpager, false)
-    }
 
-    override fun onCreateOptionsMenu(menu: Menu): Boolean {
-        menuInflater.inflate(R.menu.activity_about, menu)
-        return true
+        addMenuProvider(
+            object : MenuProvider {
+                override fun onCreateMenu(menu: Menu, menuInflater: MenuInflater) {
+                    menuInflater.inflate(R.menu.activity_about, menu)
+                }
+
+                override fun onMenuItemSelected(menuItem: MenuItem): Boolean {
+                    return when (menuItem.itemId) {
+                        R.id.show_website -> {
+                            showWebsite()
+                            true
+                        }
+                        else -> {
+                            false
+                        }
```

Comment:
Nice :) I'll test & merge soon

Just a style requests: could you please use only one line (`else -> false`) in this case?

I'm personally a fan of not putting one-liners in then/else/… within curly brackets although I know some people recommend brackets for clarity in this case